webhook/resourcesemantics/defaulting/defaulting.go

```diff
@@ -329,6 +377,48 @@ func (ac *reconciler) setUserInfoAnnotations(ctx context.Context, patches duck.J
 	return append(patches, patch...), nil
 }
 
+func (ac *reconciler) callback(ctx context.Context, gvk schema.GroupVersionKind, req *admissionv1.AdmissionRequest, patches duck.JSONPatch) (duck.JSONPatch, error) {
+	oldBytes := req.OldObject.Raw
+	newBytes := req.Object.Raw
+
+	// Setup context.
+	if len(oldBytes) != 0 {
+		ctx = apis.WithinUpdate(ctx, oldBytes)
```

Comment:
I don't think bytes is useful. We should probably store the prior unstructured resource